docs/source/releases/v2.1.rst

```diff
@@ -15,6 +15,8 @@ Welcome to Oscar 2.1.
 Compatibility
 ~~~~~~~~~~~~~
 
+Oscar 2.1 is compatible with Django 1.11, Django 2.1 and Django 2.2
```

Comment:
Please omit this for now - we'll add in when we cut a release, so that we can make sure it's up to date (Django 2.1 is now end of life and we've dropped support for it).